docs/install.md

```diff
@@ -14,147 +14,194 @@ See the License for the specific language governing permissions and
+```
 
-$ cd ./model_analyzer
+**2. Run the SDK container**
 
-$ docker build --pull -t model-analyzer .
+```
+docker run -it --gpus all \
+    -v /var/run/docker.sock:/var/run/docker.sock \
+    -v <path-to-output-model-repo>:<path-to-output-model-repo> \
+    --net=host nvcr.io/nvidia/tritonserver:22.05-py3-sdk
 ```
 
-The above command will pull all the containers that Model Analyzer needs to run.
-The Model Analyzer's Dockerfile bases the container on the latest `tritonserver`
-containers from NGC. Now you can run the container with:
+**Replacing** `<path-to-output-model-repo>` with the
+**_absolute_ _path_** to the directory where the output model repository
+will be located.
+This ensures the Triton SDK container has access to the model
+config variants that Model Analyzer creates.<br><br>
+**Important:** You must ensure the `<path-to-output-model-repo>` is identical on both sides of the mount<br><br>
 
-```
-$ docker run -it --rm --gpus all \
-      -v /var/run/docker.sock:/var/run/docker.sock \
-      -v <path-to-triton-model-repository>:<path-to-triton-model-repository> \
-      -v <path-to-output-model-repo>:<path-to-output-model-repo> \
-      --net=host model-analyzer
+**3. Add PDF support to the container**  
+Model Analyzer uses `pdfkit` for report generation. Once inside the Triton SDK container, you will need to install
+`wkhtmltopdf`:
 
-root@hostname:/opt/triton-model-analyzer# 
+```
+apt-get update && apt-get install wkhtmltopdf
 ```
 
-If you want to build Model Analyzer on the `main` branch, you need to also build the Triton
-SDK container. To build the SDK container you can refer to the
-[Build SDK Image](https://github.com/triton-inference-server/server/blob/main/docs/test.md#build-sdk-image)
-instructions. After you have built the SDK container, you can build the Model
-Analyzer's Docker image with:
+**4. Run Model Analyzer with Docker Launch Mode**  
+Be sure to use `--triton-launch-mode=docker`, when running Model Analyzer.<br><br>
 
-```
-$ git clone https://github.com/triton-inference-server/model_analyzer.git -b main
+# Alternative Installation Methods
+
+- [Specific Version with Local Launch Mode](##Specific-Version-with-Local-Launch-Mode)
+- [Main with Local Launch Mode](##Main-with-Local-Launch-Mode)
+- [Using Pip](##Using-Pip)
+- [From the Source](##From-the-Source)
 
-$ cd ./model_analyzer
+<br>
+
+## Specific Version with Local Launch Mode
+
+---
+
+This method allows you build a specific version of Model Analyzer's
+docker.  
+This installation method uses `--triton-launch-mode=local` by
+default, as all the dependencies will be available.<br><br>
+
+**1. Clone Model Analyzer's Git Repository**
 
-$ docker build -t model-analyzer --build-arg TRITONSDK_BASE_IMAGE=<name of the built SDK image> .
+```
+git clone https://github.com/triton-inference-server/model_analyzer.git -b <rXX.YY>
 ```
 
-## Triton SDK Container
+**Replacing** `<rXX.YY>` with the version of Model Analyzer you want to install<br><br>
```

Comment:
At the end mention something like :
For example, "r22.05" is the release from the end of May of 2022

Reply:
Done